consensus-engine/src/overlay/flat_overlay.rs

```diff
@@ -63,12 +63,8 @@ where
         false
     }
 
-    fn parent_committee(&self, _id: NodeId) -> crate::Committee {
-        std::iter::once(self.next_leader()).collect()
```

Comment:
oh, this was a bit of a strange behavior

Reply:
This was the quickest fix for the simulations derived from returning an empty committee. And yeah, completely bad.